### PR TITLE
Cluck rspec

### PR DESCRIPTION
**Pull request by astro-nat**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,7 +22,10 @@ gem 'jquery-rails'
 
 gem "omniauth-google-oauth2", "~> 0.2.1"
 
-gem 'pg', '~> 0.20'
+gem 'pg', '~> 0.20.0'
```
Comment (constantinahug) [inline Gemfile]: This was moved back into the production section after Zac figured out a fix

File: config/application.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module HangpersonRails
+module ExerciseApp
```
Comment (constantinahug) [inline config/application.rb]: Good catch

File: spec/models/measurements_spec.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 require 'action_controller'
-require_relative '../app/controllers/application_controller'
-require_relative '../app/controllers/measurements_controller'
+require_relative '../../app/controllers/application_controller'
```
Comment (constantinahug) [inline spec/models/measurements_spec.rb]: Can you explain why this was added? What did this change?

Comment (astro-nat) [reply spec/models/measurements_spec.rb]: Had to move this file into a spec/models folder (previously in spec), so had to change this to correct mapping

**Pull request merged**